### PR TITLE
[Feature] Support CREATE DB COMMENT (#16678)

### PR DESCRIPTION
**Pull request by xiabaike**
When create a database, add support for comment, which explains something about the database so that people who see it know what it is and what it does. CREATE DATABASE [IF NOT EXISTS] db_name COMMENT database_comment [PROPERTIES ("key"="value", ...)]; Add database's comment to the result of the SHOW CREATE DATABASE statement.

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by xiabaike** [comment]
website:
https://github.com/apache/doris-website/pull/619

File: fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java [hunk 1]
```diff
@@ -676,6 +700,7 @@ public void readFields(DataInput in) throws IOException {
         Text.readString(in);
         dbState = DbState.valueOf(Text.readString(in));
         attachDbName = Text.readString(in);
+        dbComment = Text.readString(in);
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java:703]: read comment only if version >= 132

File: fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalDatabase.java [hunk 2]
```diff
@@ -283,6 +283,16 @@ public DatabaseProperty getDbProperties() {
         return dbProperties;
     }
 
+    @Override
+    public String getDbComment() {
+        throw new NotImplementedException("getDbComment() is not implemented");
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalDatabase.java:288]: return empty string is better?

File: fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java (hunk 1 above)
Comment (xiabaike) [reply fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java:703]: Thank you for your suggestion. I will revise it.

File: fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalDatabase.java (hunk 2 above)
Comment (xiabaike) [reply fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalDatabase.java:288]: Yes, you are right.

Comment (xiabaike) [reply fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalDatabase.java]: I have modified it, please help CR again.

File: fe/fe-common/src/main/java/org/apache/doris/common/FeMetaVersion.java [hunk 3]
```diff
@@ -83,8 +83,11 @@ public final class FeMetaVersion {
     // for java-udtf add a bool field to write
     public static final int VERSION_131 = 131;
 
+    // support db comment
+    public static final int VERSION_132 = 132;
```
Comment (morrySnow) [inline fe/fe-common/src/main/java/org/apache/doris/common/FeMetaVersion.java:87]: rebase to newest master, current master's VERSION is 132, so u should use 133

Comment (xiabaike) [reply fe/fe-common/src/main/java/org/apache/doris/common/FeMetaVersion.java]: @morrySnow  Thank you for your suggestion. I have corrected it in the new pull request: https://github.com/apache/doris/pull/34625

**Pull request closed without merging**